spark/src/main/scala/geotrellis/spark/timeseries/MeanResult.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2016 Azavea
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// TODO: Do something with this?
```

Comment:
`MeanResult` was used by the old polygonal summary API and is no longer required there, but is still used by the mean operation in `geotrellis.spark.timeseries.RDDTimeSeriesMethods`.  So I put it here in the `geotrellis.spark.timeseries` package since that's the only place its currently used. Should it be moved elsewhere? If we need to refactor those methods to remove this case class for some reason, I'd prefer to leave that for a separate issue, which I'm happy to create.

Reply:
This is fine. They need to live somewhere but need to be fixed for the release. So this is +1 issue and leave it where it is for this PR.

Reply:
It turns out that I ended up needing what was effectively this class for the refactor below where each operation gets its own type alias. So I moved this type in with all the others I created, so this is addressed.